src/App.tsx

```diff
@@ -1,24 +1,414 @@
+                          <div
+                            className="modal-background
+            has-background-white-ter"
+                          />
+                        </div>
+                      </div>
+                    );
+                  }
+
+                  return <div className="loader" key={task.id} />;
+                })}
+              </>
+            )}
+
+          </section>
+
+          {todo.length > 0 && (
+            <footer className="todoapp__footer">
+              <span className="todo-count">
+                {`${numberOfActiveTodos} items left`}
+              </span>
+
+              <nav className="filter">
+                <a
+                  href="#/"
+                  className={classNames('filter__link', {
+                    selected: selectedTab === 'All',
+                  })}
+                  onClick={() => setSelectedTab(SortType.All)}
+                  role="button"
+                >
+                  All
+                </a>
+
+                <a
+                  href="#/active"
+                  className={classNames('filter__link', {
+                    selected: selectedTab === 'Active',
+                  })}
+                  onClick={() => setSelectedTab(SortType.Active)}
+                  role="button"
+                >
+                  Active
+                </a>
+
+                <a
+                  href="#/completed"
+                  className={classNames('filter__link', {
+                    selected: selectedTab === 'Completed',
+                  })}
+                  onClick={() => setSelectedTab(SortType.Completed)}
+                  role="button"
+                >
+                  Completed
+                </a>
+              </nav>
+
+              {isThereCompletedTodos ? (
+                <button
+                  type="button"
+                  className="todoapp__clear-completed"
+                  onClick={deleteCompletedTodos}
+                >
+                  Clear completed
+                </button>
+              ) : <div className="todoapp__filler-div" />}
+            </footer>
+          )}
+
+        </div>
+
+        <Error
+          message={isHidden}
+          deleteErrorMessage={deleteErrorMessage}
+          isThereIssue={isThereIssue}
+          editTodo={editTodo}
+          setIsThereIssue={setIsThereIssue}
+        />
+
+        {errorMessageField && (
+          <div
+            className="notification is-danger is-light has-text-weight-normal"
+          >
+            <button
+              type="button"
+              className="delete"
+              onClick={() => {
+                setIsDeleteingTodoAllowed(false);
+                setIsAddingTodoAllowed(false);
+                setIsEditingTodoAllowed(false);
+                setErrorMessageField(false);
+              }}
+            >
+              {null}
+            </button>
+
+            {inputValue.trim().length === 0 && (
```

Comment:
same here - you can replace this with `!inputValue.trim().length` as `0` represents `false`